### ISSUE TITLE
Accessibility (A11Y) Changes for website

### ISSUE THREAD

**Issue by lindaicing** [CubeOfCheese/Ethicli#309]
https://a11y-guidelines.orange.com/web_EN/exemples/simple-menu/simple-menu.html

On the home page:
- "BrowserExtension.dev" link on homepage fails AA contrast ratio with it's background
- "Read our pitch deck" fails AA contrast check

On The FAQ Page:
- The links in the "Where do you get your data?" section fail AA contrast ratio check
- The links in the  "We'll try to answer your commonly-asked questions!" section fail AA contrast ratio check

  Footer:  
- "Be a beta tester" text fails contrast ratio     
- Copyright text fails AA contrast

Header:
This is the only non-AA contrast concern that I really found - the "Extensions" dropdown does not follow WCAG guidelines for dropdown menus.

You need to make sure up arrow, down arrow navigate on the menu. Further, escape should close the menu, clicking outside of the menu should close it, and you'll want to make sure to use Aria Attributes to show "aria-expanded" and probably some others

**Comment by amtims** [comment]
I really like this [Accessibility Statement](https://tomboyx.com/pages/accessibility-statement) TomboyX has on their website!
